src/map/quest.h

```diff
@@ -29,8 +29,40 @@ struct block_list;
 struct config_setting_t;
 struct map_session_data;
 struct questinfo;
+struct mob_data;
 
 #define MAX_QUEST_DB (60355+1) // Highest quest ID + 1
+#define QUEST_MAPWIDE_ID 10363 // MobId handled by the client to display MapName
+#define QUEST_MOBTYPE_ID 31999 // MobId handled by the client to display Mob properties
+
+#define quest_mobtype2client(type) ((type).size | quest->mobele2client((type).ele) | quest->mobrace2client((type).race))
+#define quest_mobtypeisenabled(type) ((type).size != 0 || (type).ele_enabled || (type).race_enabled)
+
+enum quest_mobtype {
+	// Monster Races
+	TYPE_RC_DEMIHUMAN     = 0x80,
```

Comment:
In my opinion the prefix `TYPE_` is too general. I'd prefer a more specific one like we're using in other enumerations. (`SERVER_TYPE_`, `RANKTYPE_`, `MAIL_TYPE_`, etc.)
Maybe something like `QUEST_MOB_TYPE_` or `QMT_`?